account_invoice_inter_company/tests/test_inter_company_invoice.py

```diff
@@ -55,6 +55,20 @@ def test_account_invoice_inter_company(self):
             if line.product_id.company_ids:
                 line.product_id.company_ids = [(6, 0, [])]
 
+        # Check user of company B (company of destination)
+        # with which we check the intercompany product
+        self.assertNotEquals(self.user_company_b.id, 1)
```

Comment:
Not blocking, but future note - it's typically best to isolate your tests into different methods instead of creating one giant test like this. The reasoning behind this is that you will get a more clear picture of what is failing, because a failed assertion will discontinue execution of the current method and mask other errors

Reply:
@chafique-delli please use the 'msg' arg in the function
https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertEqual

Because if test fails, you'll have information of what fails. 
If not, it's undebuggable and you'll be alone to debug it :-(

And yes you may split in more atomic methods : more readeable and debuggable

Thanks

Reply:
@bealdav , @lasley , I splitted the one giant test into different methods.
@bealdav , the 'msg' arg is not necessary because the default message which appears in Odoo's log is sufficiently explicit for an Odoo developer to debug.